Logging/private/Get-LoggingDynamicParam.ps1

```diff
@@ -0,0 +1,92 @@
+<#
+.SYNOPSIS
+Creates the param used inside the DynamicParam{}-Block
+
+.DESCRIPTION
+Get-LoggingDynamicParam creates (or appends) a RuntimeDefinedParameterDictionary
+with a parameter whos value is validated through a dynamic validate set.
+
+.PARAMETER Name
+displayed parameter name
+
+.PARAMETER Level
+Constructs the validate set out of the currently configured logging level names.
+
+.PARAMETER Target
+Constructs the validate set out of the currently configured logging targets.
+
+.PARAMETER DynamicParams
+Dictionary to be appended. (Useful for multiple dynamic params)
+
+.PARAMETER Mandatory
+Controls if parameter is mandatory for call. Defaults to $true
+
+.EXAMPLE
+DynamicParam{
+    Get-LoggingDynamicParam -Name "Level" -Level -DefaultValue 'Verbose'
+}
+
+DynamicParam{
+    $dictionary = Get-LoggingDynamicParam -Name "Level" -Level
+    Get-LoggingDynamicParam -Name "Target" -Target -DynamicParams $dictionary
+}
+#>
+
+function Get-LoggingDynamicParam {
```

Comment:
Using `Get` as verb here does not seem appropriate. Get is usually for retrieving already existing data or objects. 

I believe a better name would be `New-LoggingDynamicParam`

Reply:
Thank you. I will apply the recommendations tomorrow. (no laptop till then)